babel.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  presets: [
+    [
+      "@babel/preset-env",
+      {
+        targets: {
+          node: "current"
```

Comment:
Not sure, but shouldn't the target here the browsers we support?

Reply:
for SSR projects I thought I've read to use the current node as a target since the app is precompiled on the server. but I'm also not sure

Reply:
Since the server version of node is controllable by us we don't need to specify the preset env for that, but since we need to support different browsers we should specify a lowest supported one (e.g. IE11) for preset env.
We would need node to transpile to that set of polyfills.

Reply:
```
{
  "presets": [
    [
           "@babel/preset-env",
      {
        "targets": {
          "edge": "17",
          "firefox": "60",
          "chrome": "67",
          "safari": "11.1",
        },
        "useBuiltIns": "usage",
      }
    ]
  ]
}
```

?